II.pass-tests/10-paint-ford/paint-cars.js

```diff
@@ -0,0 +1,18 @@
+function paintShop(cars) {
+    // var painted = cars.map(item => {
+    //     return Object.assign({}, item);
+    // }).map(item => {
+    //     if (item.colour === "Red") {
+    //         return item.colour = "Pink";
+    //     }
+    // })
+
+
+    //can't solve this
```

Comment:
This one isn't very well explained, sorry. The intention is that you loop through the cars to find the car with the `make` of `Ford` (hint: `.find()`) and change it's colour to `Pink`.